### ISSUE TITLE
Add a LICENSE

### ISSUE THREAD

**Issue by auscompgeek** [ncss-2015-group-4/trivia#30]
This repo is... lacking some licensing.

Currently, we're in the process of getting everyone to agree to the MIT license.

We don't appear to have anything of significance that would we might want to consider artistic licenses (such as Creative Commons) for, other than the design, but I think CC-BY-SA would be suitable for that, should the need arise.

I'll put up a list of those we're waiting on tomorrow morning when I don't have a headache from all this artificial lighting...

---

Before I forget, I should also note that Tornado is licensed under the Apache License, Version 2.0.

**Comment by paked** [comment]
Technically we just have to track people 1 by one down by their github contributions! woot

**Comment by auscompgeek** [comment]
Meh. Anyway, here's the list of people who have agreed to MIT license this project:
- [x] David (myself)
- [x] Harrison
- [x] Alex M
- [x] Ben
- [x] Tony
- [ ] @sbudiono
- [x] @JuzzyWuz
- [ ] @lachers
- [x] Maddie
- [ ] @ArethaPeethamparam
- [ ] @danielsala
- [ ] @feddielannan
- [ ] @AEthell
- [x] lkw
- [x] @KirisakiJack (John)
- [x] Trina

and for completeness, tutors:
- [x] Katie
- [x] Jack
- [x] Kenni
- [x] Alex C
- [x] Andrian

**Comment by paked** [comment]
Just for the people who don't know what the MIT License is, read below!

> The MIT License is a permissive license that is short and to the point. It lets people do anything they want with your code as long as they provide attribution back to you and don’t hold you liable. 

For a full description (and a copy of the license itself) see: http://choosealicense.com/licenses/mit/

_sidenote: http://choosealicense.com is an AWESOME resource for learning about this kind of thing, set aside some time to look through it_

**Comment by paked** [comment]
Should we start facebook PMing yet?

**Comment by paked** [comment]
@sbudiono 
@JuzzyWuz 
@lachers 
@ArethaPeethamparam 
@danielsala 
@feddielannan 
@AEthell 
@KirisakiJack 

@kennib 
@AlexanderCollins 
@andr-kun (I believe?)

**Comment by JuzzyWuz** [comment]
Yes, sure! Sorry for the late support but yeah, of course, I agree to the license.

**Comment by kennib** [comment]
I'm fine with any licence you choose. Keep up the good work!

**Comment by AlexanderCollins** [comment]
Hey,
Sorry for the late reply also, I also agree to the licence!

Cheers [?]
Alex.

**Comment by andr-kun** [comment]
Hi everyone,

I agree to the license and am happy with whichever license is chosen.

Andrian

**Comment by auscompgeek** [comment]
I just realised that our trivia questions could do with some Creative Commons licensing. Unfortunately, I have no idea who wrote the newer questions. Bummer.
